src/common/enum_util.cpp

```diff
@@ -2580,14 +2580,18 @@ const char* EnumUtil::ToChars<JoinType>(JoinType value) {
 		return "INNER";
 	case JoinType::OUTER:
 		return "FULL";
-	case JoinType::SEMI:
+	case JoinType::LEFT_SEMI:
 		return "SEMI";
-	case JoinType::ANTI:
+	case JoinType::LEFT_ANTI:
 		return "ANTI";
 	case JoinType::MARK:
 		return "MARK";
 	case JoinType::SINGLE:
 		return "SINGLE";
+	case JoinType::RIGHT_SEMI:
+		return "SEMI";
+	case JoinType::RIGHT_ANTI:
+		return "ANTI";
```

Comment:
Is it not a problem that both LEFT_ANTI and RIGHT_ANTI convert to "ANTI"?

Reply:
It would be a problem if the converted to "LEFT ANTI" and "RIGHT ANTI". When verification is enable we "ToString" the statement, and we can't parse "LEFT SEMI/ANTI" SQL. In the `ParamsToString()` function I add the "LEFT" and "RIGHT" back. This way we can see which one is used in the plans

Reply:
It is a problem, actually. At the top of the `enum_util.cpp` file, it says:
```c++
//-------------------------------------------------------------------------
// This file is automatically generated by scripts/generate_enum_util.py
// Do not edit this file manually, your changes will be overwritten
// If you want to exclude an enum from serialization, add it to the blacklist in the script
//
// Note: The generated code will only work properly if the enum is a top level item in the duckdb namespace
// If the enum is nested in a class, or in another namespace, the generated code will not compile.
// You should move the enum to the duckdb namespace, manually write a specialization or add it to the blacklist
//-------------------------------------------------------------------------
```
You should not edit the file manually, and use `python scripts/generate_enum_util.py` instead.

This will change the method to return `"RIGHT_SEMI"` and `"RIGHT_ANTI"`.